app/views/VerifyPrivateTraderDetailView.scala.html

```diff
@@ -66,11 +67,14 @@
 
         @caption(messages("caption.applicant"))
         @heading(messages("traderDetails.private.heading", details.EORINo))
-        @paragraph(Html(messages("traderDetails.private.p1")))
+
+        <br>
+        @warning(content=Html(messages("traderDetails.private.p1")))
         @paragraph(Html(messages("traderDetails.private.p2")))
 
 
-        @h3(messages("traderDetails.private.h3.proceed"))
+        @subheading(messages("traderDetails.private.h3.proceed"))
```

Comment:
Strangely it looks the same as h3 and it's required to be in order h1...h2..h3... etc to pass accessibility final checks. Perhaps leave off on merging until monday when I can sanity check this again because I want to make sure this makes sense.